src/style.css

```diff
@@ -1,3 +1,89 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use the '*' selector (it impacts performance). Set styles only for elements that require them. Instead of zeroing out margins and paddings for all elements, set them specifically where needed.